express/api_routes.js

```diff
@@ -30,8 +33,12 @@ function setupProxyRoutes() {
         }
     });
     
-    apiRoutes.use(sessionMiddleware(true), async (req, res, next) => {
+    apiRoutes.use(["/", "/:cluster_id"], sessionMiddleware(extension, true), async (req, res, next) => {
```

Comment:
["/", "/:cluster_id"] should be swapped ["/:cluster_id", "/"]

Reply:
done